### PR TITLE
add multiple languages to sidebar with javascript

### PR DESCRIPTION
**Pull request by Tony931023**
**Add multiple languages to sidebar with javascript**

@Kitzunu check and test to see if in this way the wiki sidebar allows you to switch between languages, for the implementation of the pages in Spanish
### Locally tested and it works

### CODE REVIEW

File: _includes/azerothcore/sidebar.html [hunk 1]
```diff
@@ -1,57 +1,137 @@
...
+	
+    <script language= "JavaScript">
+
+	function cambiar_a(lenguaje)
+        {            
+            document.getElementById('es').style.display= (lenguaje=='español')? 'inline' : 'none';
+            document.getElementById('en').style.display= (lenguaje=='en')? 'inline' : 'none';
+        }
+	
+    </script>
```
Comment (Helias) [inline _includes/azerothcore/sidebar.html:12]:
```suggestion
    <script>
	function changeLang(language)
	{
            document.getElementById('es').style.display = (language == 'español') ? 'inline' : 'none';
            document.getElementById('en').style.display = (language == 'en') ? 'inline' : 'none';
	}
    </script>
```

File: _includes/azerothcore/sidebar.html [hunk 2]
```diff
@@ -1,57 +1,137 @@
...
+    
+    <div id='language'>
```
Comment (Helias) [inline _includes/azerothcore/sidebar.html:14]:
```suggestion
    <div id="language">
```

File: _includes/azerothcore/sidebar.html [hunk 3]
```diff
@@ -1,57 +1,137 @@
...
+		<ol>
+            <b style="cursor: pointer;" onclick="cambiar_a('español');"><img src="https://github.com/azerothcore/wiki/blob/test/images/es.png" alt="Español"></b>
+            <b style="cursor: pointer;" onclick="cambiar_a('en');"><img src="https://github.com/azerothcore/wiki/blob/test/images/gb.png" alt="Ingles"></b>
```
Comment (Helias) [inline _includes/azerothcore/sidebar.html:17]:
```suggestion
            <b style="cursor: pointer;" OnClick="changeLang('español');"><img src="https://github.com/azerothcore/wiki/blob/test/images/es.png" alt="Español"></b>
            <b style="cursor: pointer;" OnClick="changeLang('en');"><img src="https://github.com/azerothcore/wiki/blob/test/images/gb.png" alt="Ingles"></b>
```

**Comment by Helias** [comment]
Not sure if this is the best way to change language, I would prefer to keep in a JSON dictionary all the keywords en/es and change only an index to change the language, so reduce the redundant code

**Comment by Tony931023** [comment]
@Helias  with JSON it would be a good idea, actually I would not know how to implement it like that, you have to add the libraries of https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js, I think javascript it is faster in this case.

The Wiki would look like this for the language change in the sidebar only, and in the other pages it would be something like @Kitzunu  did on the home page.

**In Spanish**

es

**in English**
in

File: _includes/azerothcore/sidebar.html [hunk 4]
```diff
@@ -1,57 +1,137 @@
...
-    <ul>
-        <li><a target="_blank" href="http://www.azerothcore.org/pages/sunwell.pl/">Original SC Description</a></li>
-    </ul>
+	
+    <script language= "JavaScript">
+
+	function changeLang(lenguaje)
+        {            
+            document.getElementById('es').style.display= (lenguaje=='español')? 'inline' : 'none';
+            document.getElementById('en').style.display= (lenguaje=='en')? 'inline' : 'none';
```
Comment (Kitzunu) [inline _includes/azerothcore/sidebar.html:9]:
```suggestion
	function changeLang(language)
        {
            document.getElementById('en').style.display= (language=='en')? 'inline' : 'none';
            document.getElementById('es').style.display= (language=='es')? 'inline' : 'none';
```

File: _includes/azerothcore/sidebar.html [hunk 5]
```diff
@@ -1,57 +1,137 @@
...
+        }
+	
+    </script>
+    
+    <div id="language">
+		<ol>
+            <b style="cursor: pointer;" onclick="changeLang('español');"><img src="https://github.com//azerothcore/wiki/blob/acbf3f05e90da007939ae32c8e9ec940f2e622ec/images/es.png?raw=true" alt="Español"></b>
+            <b style="cursor: pointer;" onclick="changeLang('en');"><img src="https://github.com/azerothcore/wiki/blob/acbf3f05e90da007939ae32c8e9ec940f2e622ec/images/gb.png?raw=true"" alt="Ingles"></b>
```
Comment (Kitzunu) [inline _includes/azerothcore/sidebar.html:17]:
```suggestion
            <b style="cursor: pointer;" onclick="changeLang('en');"><img src="https://github.com/azerothcore/wiki/blob/acbf3f05e90da007939ae32c8e9ec940f2e622ec/images/gb.png?raw=true" alt="English"></b>
            <b style="cursor: pointer;" onclick="changeLang('es');"><img src="https://github.com/azerothcore/wiki/blob/acbf3f05e90da007939ae32c8e9ec940f2e622ec/images/es.png?raw=true" alt="Spanish"></b>
```

File: _includes/azerothcore/sidebar.html [hunk 6]
```diff
@@ -1,57 +1,137 @@
...
+		</ol>
+	</div>
+	
+    <div id="es"  style="display: none;"> 
+	   
+        <b>How-To ( branch master )</b> <a href="https://github.com/azerothcore/wiki/edit/master/_includes/azerothcore/sidebar.html">Edit</a>
```
Comment (Kitzunu) [inline _includes/azerothcore/sidebar.html:23]: put english at top

**Review by Kitzunu — CHANGES_REQUESTED** [review]
Is there a way to split the English/Spanish versions into 2 separate files and importing them into this one?

**Comment by Kitzunu** [comment]
Is there a way to split the English/Spanish versions into 2 separate files and importing them into this one?

**Comment by Tony931023** [comment]
Maybe if the sidebar shows what is inside the <div class = "sidebar-wrapper"> tags, maybe doing something with JSON as a structure that will show the content according to the user's language. or to call another page and enclose it inside the sidebar with a structure.

  **I do not know if I explained well but more or less it is an idea**

**Comment by Kitzunu** [comment]
I dont know web stuff well enough. All I thought of was if all the data could be separated, otherwise that file will be big as fuck

**Comment by Tony931023** [comment]
I do not think that for now this is the solution that I find most viable to translate the sidebar, also I do not think that for now another language other than Spanish is implemented in the wiki

**Comment by Kitzunu** [comment]
Did you test the changes again after updating the names?

**Comment by Tony931023** [comment]
> 
> 
> Did you test the changes again after updating the names?

yes

**Pull request merged**